### PR TITLE
[bitnami/wavefront-hpa-adapter] Add tests and publishing using VIB

### PR DESCRIPTION
**Pull request by joancafom**
### Description of the change

The main objective of this PR is to publish our Bitnami Wavefront HPA Adapter Helm Chart using VMware Image Builder. In order to do that, several changes are included:

- Adding the asset to the publishing workflow
- Increasing the existing test coverage of the asset by adding Ginkgo tests.

### Benefits

- Ensuring higher quality of the Helm charts.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could show to be potentially flaky. 

### Applicable issues

NA

### Additional information

Tested in https://github.com/joancafom/charts/pull/95

### Checklist

- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/wavefront-hpa-adapter/vib-publish.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/wavefront-hpa-adapter"
         },
-        "runtime_parameters": "InNlcnZpY2UiOgogICJwb3J0cyI6CiAgICAiaHR0cHMiOiA0NDMKICAidHlwZSI6ICJMb2FkQmFsYW5jZXIiCiJ3YXZlZnJvbnQiOgogICJ0b2tlbiI6ICJmYWtlIgogICJ1cmwiOiAiaHR0cHM6Ly9mYWtlLndhdmVmcm9udC5jb20i",
+        "runtime_parameters": "Y29udGFpbmVyUG9ydHM6CiAgaHR0cHM6IDY0NDQKd2F2ZWZyb250OgogIHVybDogaHR0cHM6Ly9mYWtlLndhdmVmcm9udC5jb20KICB0b2tlbjogRkFLRV9BUElfVE9LRU4Kc2VydmljZToKICB0eXBlOiBMb2FkQmFsYW5jZXIKICBwb3J0czoKICAgIGh0dHBzOiA0NDMKcmJhYzoKICBjcmVhdGU6IHRydWUKYXBpU2VydmljZToKICBjcmVhdGU6IHRydQpleHRyYURlcGxveToKLSBhcGlWZXJzaW9uOiBhdXRvc2NhbGluZy92MmJldGExCiAga2luZDogSG9yaXpvbnRhbFBvZEF1dG9zY2FsZXIKICBtZXRhZGF0YToKICAgIG5hbWU6IHZpYi1ocGEtY3VzdG9tLW1ldHJpY3MKICBzcGVjOgogICAgbWluUmVwbGljYXM6IDEKICAgIG1heFJlcGxpY2FzOiA1CiAgICBtZXRyaWNzOgogICAgLSB0eXBlOiBFeHRlcm5hbAogICAgICBleHRlcm5hbDoKICAgICAgICBtZXRyaWNOYW1lOiB2aWIudm0uY3B1CiAgICAgICAgdGFyZ2V0QXZlcmFnZVZhbHVlOiAxODAwbQogICAgc2NhbGVUYXJnZXRSZWY6CiAgICAgIGFwaVZlcnNpb246IGFwcHMvdjEKICAgICAga2luZDogRGVwbG95bWVudAogICAgICBuYW1lOiB3YXZlZnJvbnQtaHBhLWFkYXB0ZXI=",
```
Comment (FraPazGal) [inline .vib/wavefront-hpa-adapter/vib-publish.json:25]: You have a typo on the params:
```
apiService:
  create: tru
```

**Review by FraPazGal — COMMENTED** [review]
Thanks for the PR @joancafom, added a minor comment

File: .vib/wavefront-hpa-adapter/vib-publish.json (hunk 1 above)
Comment (joancafom) [reply .vib/wavefront-hpa-adapter/vib-publish.json]: Oh, it is "tru" I have a typo 😆 Fixed it, thanks!

**Review by agomezmoron — APPROVED** [review]

**Review by FraPazGal — APPROVED** [review]
LGTM, thanks for the PR!

**Pull request merged**